friends.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+const friendSchema = mongoose.Schema({
+    firstName : {type:String,required:true},
+    lastName : {type:String,required:true},
+    age : {type :Number,min : 1, max: 120},
```

Comment:
Very nice! I like that you moved the validation to the schema layer, instead of the router.